recipes/tensap/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.0" %}
+
+package:
+  name: tensap
+  version: {{ version }}
+
+source:
+  #url: https://github.com/anthony-nouy/tensap/archive/v{{ version }}.tar.gz
+  #sha256: e0828ad9b2a330d996b2ce4a445980dd7bc22e79573abb7b36ff0920779b8a9d
+
+  # we need a few commits after v1.0 tag for the setup.py
+  git_url: https://github.com/anthony-nouy/tensap
+  git_rev: a5d26f4de04f2868f03d5a7932ca64d9d127c93a
```

Comment:
Please build from source, not git

Reply:
@scopatz I would, but as mentioned in the comment above, we need a few more commits past the v1.0 tag that added support for setup.py

Reply:
Can you do this for an old release for now? Or add a patch file here?

Reply:
using v1.1